keyboards/wuque/ikki68_aurora/info.json

```diff
@@ -0,0 +1,86 @@
+{
```

Comment:
This doesnt render correctly on configurator (previewed with `ctrl + shift + i`)

![image](https://user-images.githubusercontent.com/16520359/107162289-01bd5f00-699a-11eb-8c1c-99da3400736a.png)

Reply:
I corrected my mistake, but youcan see its key is splited. I did my best.
https://cdn.shopify.com/s/files/1/0403/9809/4489/t/6/assets/pf-8e74bde4--finallayout.jpg?v=1612035114

Reply:
For that, you can have multiple different layout macros defined, and included in the info.json.  This allows you to select from different layouts. And that's also why we recommend a `LAYOUT_all`, that has all possible key positions enabled.  A good example of this is the dz60 keyboard.

Reply:
The main problem is that this is a hot-swappable keyboard that can change the layout, I need to keep unneeded key values in the same layout. It contains splittable spaces, splittable shift etc. It makes him look weird. 
![image](https://user-images.githubusercontent.com/72414103/121662797-6c002080-caae-11eb-93b4-7ce47f31a5e2.png)

Reply:
if that's the case, then you should have multiple layouts, or have a `LAYOUT_all` that has all possible positions. 

You can look at /layouts/defaults/ for references,  or the dz60, for how to implement multiple layouts.